db/re/skill_db.yml

```diff
@@ -27266,6 +27264,7 @@ Body:
       - Level: 5
         Time: 6000
     AfterCastActDelay: 1000
+    Duration: 18000
```

Comment:
Github glitches out for me at the end of the file so I can't see the context, but is this really "Duration" and not "Duration1" or "Duration2"?